migration_tools.drush.inc

```diff
@@ -248,6 +263,66 @@ function drush_migration_tools_mt_migrate_file_folders($organization) {
 }
 
 /**
+ * Drush command callback for migration-tools-html-file-list.
+ *
+ */
+function drush_migration_tools_html_file_list($directory) {
+  // Set up options
+  $format = drush_get_option('format', 'txt');
+  $outfile = drush_get_option('outfile');
+  $mask = '/.*\.htm(l)?$/i';
+
+  switch ($format) {
+    case "csv":
+      $delimiter = ", ";
+      $newline = "\n";
+      break;
+
+    default:
+      $delimiter = "\t";
+      $newline = "\n";
+  }
+
+  // Fetch file list
+  $options = array();
+  $files = file_scan_directory($directory, $mask, $options);
+
+  // Generate output
+  $output = "Path" . $delimiter . $newline;
+  foreach ($files as $file) {
+    $output .= $file->uri . $delimiter;
+    $output .= $newline;
+  }
+
+  // Attempt to write results to file if requested, otherwise output to terminal.
+  if ($outfile) {
+    $scanned_directory = trim(basename($directory), ".");
+    $outfile_path = $scanned_directory . "." . $format;
+
+    $handle = fopen($outfile_path, "w");
+
+    if(!$handle) {
+      drush_log("There was an error opening {$outfile_path} for writing, verify the path is writable and try again.", 'error');
+    }
+    else {
```

Comment:
After you message that there was an error, you can output the file to the screen.  

That way if we run it on a prod type server where there is no file write access, we can still get something useful in terminal for copy and paste.